$OpenBSD: patch-fastecdsa_tests_test_whycheproof_vectors_py,v 1.2 2021/07/14 22:15:23 tb Exp $

Use the JSON files from security/wycheproof-testvectors instead of fetching
them from GitHub.

Index: fastecdsa/tests/test_whycheproof_vectors.py
--- fastecdsa/tests/test_whycheproof_vectors.py.orig
+++ fastecdsa/tests/test_whycheproof_vectors.py
@@ -25,7 +25,9 @@ class TestWycheproofEcdsaVerify(TestCase):
     @staticmethod
     def _get_tests(url):
         try:
-            test_raw = urlopen(url).read()
+            url = url.replace('https://raw.githubusercontent.com/google/wycheproof/master/', '${LOCALBASE}/share/wycheproof/')
+            with open(url) as f:
+                test_raw = f.read()
             test_json = loads(test_raw)
             return test_json["testGroups"]
         except (JSONDecodeError, URLError) as error:
